R/catmaid_connection.R

```diff
@@ -441,12 +445,12 @@ catmaid_connection_fingerprint<-function(conn) {
 #' @seealso \code{\link{catmaid_login}}
 #' @export
 catmaid_connection_setenv<-function(conn=NULL, ...) {
-  conn=catmaid_login(conn, ...)
+  if (is.null(conn)){conn=catmaid_login(conn, ...)} #only do the connection if I supply a null..
```

Comment:
`catmaid_login` looks after checking the value of conn already – so just call it as before i.e. revert this change.
```suggestion
 conn=catmaid_login(conn, ...)
```

Reply:
I have updated this in the latest commit in line 446.